Podcast/app/src/main/java/br/ufpe/cin/if710/podcast/ui/MainActivity.java

```diff
@@ -106,17 +137,23 @@ protected void onPostExecute(List<ItemFeed> feed) {
             //atualizar o list view
             items.setAdapter(adapter);
             items.setTextFilterEnabled(true);
-            /*
+
             items.setOnItemClickListener(new AdapterView.OnItemClickListener() {
+
                 @Override
                 public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
                     XmlFeedAdapter adapter = (XmlFeedAdapter) parent.getAdapter();
                     ItemFeed item = adapter.getItem(position);
-                    String msg = item.getTitle() + " " + item.getLink();
-                    Toast.makeText(getApplicationContext(), msg, Toast.LENGTH_SHORT).show();
+                    Intent intent = new Intent(MainActivity.this, EpisodeDetailActivity.class);
+
+                    intent.putExtra(MainActivity.CLICKED_TITLE, item.getTitle().replaceAll("\\s+", " "));
+                    intent.putExtra(MainActivity.CLICKED_DESCRIPTION, item.getDescription().replaceAll("\\s+", " "));
+                    intent.putExtra(MainActivity.CLICKED_PUBDATE, item.getPubDate());
+                    intent.putExtra(MainActivity.CLICKED_DOWNLOADLINK, item.getDownloadLink());
+                    startActivity(intent);
                 }
```

Comment:
Implementing _ setOnItemClickListener_ in order to display the details of an episode after clicking, saving the fields which _EpisodeDetailActivity_  expects